docs/getting-started.rst

```diff
@@ -1,59 +1,106 @@
+---------------
 Getting started
 ---------------
 
-Requirements:
 
-- ``docker`` installed and running
-- ``Node.js`` >= 7.6 - ``async/await`` support
 
-1. Download repositories: gardener-server, gardener-smart-contracts
+
+Repositories
+------------
+
+Gardener consists of three repositories, two main ones: gardener-smart-contracts which holds smart contracts, and gardener-server, which is responsible for fetching data from third party data sources.
+The third one, gardener-monitor is optional and it helps visualizing requests.
+
+1. Gardener server
+::
+
+  git clone https://github.com/EspeoBlockchain/gardener-server.git
+
+2. Gardener smart contracts
+::
+
+  git clone https://github.com/EspeoBlockchain/gardener-smart-contracts.git
+
+3. Gardener monitor (optional)
+::
+
+  git clone https://github.com/EspeoBlockchain/gardener-monitor.git
+
+
+Running blockchain - Server
+------------------
+
+Before we will get information from external sources to our blockchain, we have to run it first. Let's use ganache for it.
+First of all, copy server variables from template:
 ::
 
-   git clone https://github.com/EspeoBlockchain/gardener-server.git
-   git clone https://github.com/EspeoBlockchain/gardener-smart-contracts.git
+  make copy-env
+
+After that, we can run our blockchain. Let's use ganache for it:  
+::
 
+  make ganache
 
-2. Copy smart contracts variables from template
+If you see this information:
 ::
 
-   cd gardener-smart-contracts
-   make copy-env
+  Creating test-blockchain ... done
 
-3. Copy server variables from template
+That means you have created test blockchain successfully. You can verify its status using:
 ::
 
-   cd ../gardener-server
-   make copy-env
+  docker ps
+
+Running blockchain - Monitor (optional)
+------------------------
+::
+
+  cd ../gardener-monitor
+  cp .env.tpl .env
+  npm install
+  npm start
+
+Running blockchain - Smart contracts
+-------------------
 
-4. Run test blockchain
+After starting blockchain, we need to copy our smart contracts variables from template. 
+Make sure that you are in gardener-smart-contracts directory, then:
 ::
 
-   make ganache
-   docker ps
+  make copy-env
 
-5. Install smart contracts dependencies
+Now, we are going to install dependencies that Gardener smart contract relies on. When you are in gardener-smart-contracts directory, then:
```

Comment:
We've asserted in the previous step that the user should be in the correct directory.